pkgs/top-level/all-packages.nix

```diff
@@ -13387,8 +13390,12 @@ let
 
   vym = callPackage ../applications/misc/vym { };
 
-  w3m = callPackage ../applications/networking/browsers/w3m {
+  w3m = callPackage ../applications/networking/browsers/w3m { };
+
+  w3m-pure = w3m.override {
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/10851#discussion_r45405883 for a detailed explanation.